modules/rpc/rpc.js

```diff
@@ -100,7 +100,7 @@ exports.call = function(method, params, callback) {
       case 'ECONNREFUSED':
         callback({
           status: 502,
-          message: 'Connection Refused',
+          message: 'Daemon not connected, retrying connection',
```

Comment:
Is this change required ?

Reply:
Yeah, I think it better describes the the error to the user as connection is not likely to be actually refused